clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -197,18 +197,20 @@
 
 public class RequestResponseTest {
 
+    private UnknownServerException unknownServerException = new UnknownServerException("secret");
```

Comment:
nit: I would add a small comment to explain why we do this here.

Reply:
@dajac Thanks for the review, added comment.